nrt/data/SVarC/SVarC_4_ModeChange/SVarC.dyd

```diff
@@ -13,17 +13,18 @@
 -->
 <dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
   <!-- SVarC and infinite bus-->
-  <dyn:modelicaModel id="Test">
-    <dyn:unitDynamicModel id="busInf" name="Dynawo.Electrical.Buses.InfiniteBus" parFile="SVarC.par" parId="1"/>
-    <dyn:unitDynamicModel id="line" name="Dynawo.Electrical.Lines.Line" parFile="SVarC.par" parId="2"/>
+  <dyn:blackBoxModel id="busInf" lib="InfiniteBus" parFile="SVarC.par" parId="1"/>
+  <dyn:blackBoxModel id="line" lib="Line" parFile="SVarC.par" parId="2"/>
+  <dyn:modelicaModel id="staticVarCompensator">
```

Comment:
Why didn't you use a blackBoxModel here ? With the proper continuous optional extvar it's possible no ?

Reply:
There is an initConnect in this modelicaModel, which bars me from using the StaticVarCompensator preassembled model.

Reply:
Well you can give Value0 in the .par accordingly to avoid this INIT connect

Reply:
URef0 is calculated in SVarCStandard_INIT.mo with the following equation :

_URef0 = (U0Pu - Lambda*Q0Pu*SystemBase.SnRef/SNom) * UNom;_

I would rather use this equation than set URef0 in the .par.

Reply:
And it is used in the general case when no step is used. But just for this test case, as we want to avoid unitDynamicModel, let's give Value0 in the .par

Reply:
Agreed and done